app/app.py

```diff
@@ -59,8 +65,20 @@ def process_hook(api, data):
 
 
 def trigger_openshift_ci_job(job):
-    # TODO: Call openshift-ci to trigger the job
+    openshift_ci_token_env_variable_str = "OPENSHIFT_CI_TRIGGER_TOKEN"
```

Comment:
Token can be either os env or configured in the yaml

Reply:
moved it to the config